docs/commands/index.md

```
@@ -0,0 +1,65 @@
+  --lower-time-bound "2024-01-01T00:00:00" \
+  --upper-time-bound "2024-01-31T23:59:59" \
+  --batch-size 200 \
+  --async_mode
+```
+
+### Django Admin
+
+The command is available in the Django admin interface:
+
+1. Go to `/admin/core/run-indexing/`, you arrive at the "Run Indexing Command" page
+3. Fill in the form with the desired parameters
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 2; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)